alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,58 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, in either the platform Prometheus or a user-defined monitoring
+Prometheus pods..
+
+## Impact
+
+Prometheus samples in remote storage might be missing.
```

Comment:
Maybe add
`Depending on the remote_write pipeline configuration, prometheus memory usage might increase while pending samples are queued.`

Reply:
Done